### PR TITLE
Infrastructure Versioning Update

### PR DESCRIPTION
**Pull request by OSIErickRC**
- Added new docs based on version (V1 & V1-preview)
- Updated relevant .yml & index files

### CODE REVIEW

File: Documentation/Access_Control.md [hunk 1]
```diff
@@ -2,39 +2,26 @@
...
 
+Owners are granted access for all operations regardless of the contents of the ACL. Not all entities in the OCS system support role-based access control at this time, but the list will quickly grow and currently includes Namespaces and several unreleased entities.
 
-Access Control Lists
---------------------
+## Access Control Lists
 
-Access Control Lists (ACLs) contain sets of Access Control Entries (ACEs) each with a trustee (reference to an identity, such 
-as a role, user, or application), AccessType, and AccessRights. 
+Access Control Lists (ACLs) contain sets of Access Control Entries (ACEs) each with a trustee (reference to an identity, such as a role, user, or application), AccessType, and AccessRights.
```
Comment (osimarkrogers) [inline Documentation/Access_Control.md]: add comma after '(ACEs)'

File: Documentation/Management/Account_ClientRole_1.md [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+
+### Parameters
+
+```csharp
+[Required]
+[FromRoute]
+string tenantId
+```
+
+The identifier for the account in which the client belongs to.
```
Comment (osimarkrogers) [inline Documentation/Management/Account_ClientRole_1.md]: Change to 'The identifier of the account to which the client belongs'

File: Documentation/Management/Account_Role_1-preview.md [hunk 3]
```diff
@@ -187,7 +159,7 @@ Authorized for Account Administrators of the specified account.
 
 | Status Code | Return Type | Description | 
  | --- | --- | ---  | 
-| 200 | Role | Returns the `Role` with specified Id roleId. | 
+| 200 | Role | Returns the `Role` with with specified Id roleId. |
```
Comment (osimarkrogers) [inline Documentation/Management/Account_Role_1-preview.md]: 'with' appears to be intentionally duplicated here?

File: Documentation/Management/Account_Role_1.md [hunk 4]
```diff
@@ -0,0 +1,273 @@
+---
+uid: AccountRole_1
+---
+
+# Roles (Version 1)
+
+A `Role` is used to manage access within an OSIsoft Cloud Services (OCS) account. By default, users have the Account Member role. There
```
Comment (osimarkrogers) [inline Documentation/Management/Account_Role_1.md]: I would change 'By default....' to 'All users are assigned the Account Member role by default.'

File: Documentation/Access_Control.md [hunk 5]
```diff
@@ -2,39 +2,26 @@
...
-Role-based access control 
-=========================
+# Role-based access control 
 
-Within OCS, role-based access control to entities such as Namespaces, Streams, and so on, is managed using an Access Control 
-List (ACL) and an Owner identity assigned to each entity. ACLs control access to entities based on their OCS Roles. Owners 
-are granted access for all operations regardless of the contents of the ACL. Not all entities in the OCS system support 
-role-based access control at this time, but the list will quickly grow and currently includes Namespaces and several unreleased 
-entities.
+Within OCS, role-based access control to entities such as Namespaces, Streams, and so on, is managed using an Access Control List (ACL) and an Owner identity assigned to each entity. ACLs control access to entities based on their OCS Roles.
```
Comment (osimarkrogers) [inline Documentation/Access_Control.md]: Change to 'entities such as Namespaces and Streams, is managed....'

File: Documentation/Management/Account_ServiceBlog.md [hunk 6]
```diff
@@ -45,7 +55,7 @@ Any identity, including anonymous, can retrieve service blog entries.
 
 | Status Code | Return Type | Description | 
  | --- | --- | ---  | 
-| 200 | ([SdsServiceBlogEntry], int32) | Returns a response object including a list of blog entries and an int with the total number of entries. | 
+| 200 | ([SdsServiceBlogEntry], int32) | Returns an response object including a list of blog entries and an int with the total number of entries. |
```
Comment (osimarkrogers) [inline Documentation/Management/Account_ServiceBlog.md]: Should be 'a response object'

File: Documentation/Management/Account_ServiceBlog.md [hunk 7]
```diff
@@ -79,7 +90,7 @@ Any identity, including anonymous, can retrieve service blog entries.
 
 | Status Code | Return Type | Description | 
  | --- | --- | ---  | 
-| 200 | SdsServiceBlogEntry | Returns a response object including the specified blog entry. | 
+| 200 | SdsServiceBlogEntry | Returns an response object including the specified blog entry. |
```
Comment (osimarkrogers) [inline Documentation/Management/Account_ServiceBlog.md]: Should be 'a response object'

File: Documentation/Management/Account_TenantFeatureState.md [hunk 8]
```diff
@@ -28,13 +28,14 @@ For HTTP requests and responses, the TenantFeatureState object has the following
 
 ## `Get All Tenant Feature States`
 
-Retrieves all `FeatureStates` for the specified `Tenant`.
+Retrieves all `FeatureState` s for the specified `Tenant`.
```
Comment (osimarkrogers) [inline Documentation/Management/Account_TenantFeatureState.md]: Not sure what to do here - - is 'FeatureState' the name of an object?  If trying to make plural, probably better to phrase it as 'Retrieves all instances of 'FeatureState' for the specified 'Tenant'

File: Documentation/Management/Account_TenantFeatureState.md [hunk 9]
```diff
@@ -55,9 +56,9 @@ Authorized for Account Members of the specified account.
 
 | Status Code | Return Type | Description | 
  | --- | --- | ---  | 
-| 200 | [FeatureState] | Returns a list of `FeatureStates` for the specified account. | 
-| 400 | Nothing is returned | Unable to retrieve `FeatureStates` due to missing or invalid input. | 
-| 403 | Nothing is returned | Unauthorized to retrieve `FeatureStates` for the specified account. | 
+| 200 | [FeatureState] | Returns a list of `FeatureState` s for the specified account. |
```
Comment (osimarkrogers) [inline Documentation/Management/Account_TenantFeatureState.md]: Change to 'Returns a list of 'FeatureState' instances for....'

File: Documentation/Management/Account_TenantFeatureState.md [hunk 10]
```diff
@@ -55,9 +56,9 @@ Authorized for Account Members of the specified account.
 
 | Status Code | Return Type | Description | 
  | --- | --- | ---  | 
-| 200 | [FeatureState] | Returns a list of `FeatureStates` for the specified account. | 
-| 400 | Nothing is returned | Unable to retrieve `FeatureStates` due to missing or invalid input. | 
-| 403 | Nothing is returned | Unauthorized to retrieve `FeatureStates` for the specified account. | 
+| 200 | [FeatureState] | Returns a list of `FeatureState` s for the specified account. | 
+| 400 | Nothing is returned | Unable to retrieve `FeatureState` s due to missing or invalid input. |
```
Comment (osimarkrogers) [inline Documentation/Management/Account_TenantFeatureState.md]: Change to 'Unable to retrieve 'FeatureState' instances due to ....'

File: Documentation/Management/Account_TenantFeatureState.md [hunk 11]
```diff
@@ -55,9 +56,9 @@ Authorized for Account Members of the specified account.
 
 | Status Code | Return Type | Description | 
  | --- | --- | ---  | 
-| 200 | [FeatureState] | Returns a list of `FeatureStates` for the specified account. | 
-| 400 | Nothing is returned | Unable to retrieve `FeatureStates` due to missing or invalid input. | 
-| 403 | Nothing is returned | Unauthorized to retrieve `FeatureStates` for the specified account. | 
+| 200 | [FeatureState] | Returns a list of `FeatureState` s for the specified account. | 
+| 400 | Nothing is returned | Unable to retrieve `FeatureState` s due to missing or invalid input. | 
+| 403 | Nothing is returned | Unauthorized to retrieve `FeatureState` s for the specified account. |
```
Comment (osimarkrogers) [inline Documentation/Management/Account_TenantFeatureState.md]: Change to 'Unable to retrieve 'FeatureState' instances for the....'

File: Documentation/Management/Management_Overview.md [hunk 12]
```diff
@@ -1,28 +1,41 @@
 ---
-uid: ManagementOverview
+uid: AccountManagementOverview
 ---
 
 # Account Management
 
-The Account Management API allows users to create, manage, and administer OSIsoft Cloud Services (OCS) accounts by assigning users, roles, and namespaces to their tenants. Through the API, users can manage access control used the OCS infrastructure. Generally, only administrators access the Account Management features. Administration functionality is also available from the OCS portal.
+The Account Management API allows users to create, manage, and administer OSIsoft Cloud Services (OCS) accounts by assigning users, roles, and namespaces to their tenants. Through the API, users can manage access control used the OCS infrastructure. Generally, only administrators access the Account Management features. Administration functionality is also available from the OCS portal. Below is an index of the Account Management API:
```
Comment (osimarkrogers) [inline Documentation/Management/Management_Overview.md]: Change to 'Through the API, users can manage access control to use the OCS infrastructure.'

**Comment by OSIErickRC** [comment]
Remaining edits resolved.

**Pull request merged**